scripts/e2e-test.sh

```diff
@@ -16,19 +16,76 @@
 
 set -eEuo pipefail
 
+PROGNAME="$(basename $0)"
 ROOT="$(cd "$(dirname "$0")/.." &>/dev/null; pwd -P)"
 
-function main() {
+function smoke() {
     # PROW_JOB_ID is an env var set by prow, use project for prow when it's in prow
     if [[ -z "${PROJECT_ID:-}" && -n "${PROW_JOB_ID:-}" ]]; then
-        PROJECT_ID="$(boskos_acquire)"
+        PROJECT_ID="$(boskos_acquire verification-smoke-project)"
         trap "boskosctl_wrapper release --name \"${PROJECT_ID}\" --target-state dirty >&2" EXIT
         export PROJECT_ID
     fi
 
     ${ROOT}/scripts/terraform.sh smoke
 }
 
+function incremental() {
+   # PROW_JOB_ID is an env var set by prow, use project for prow when it's in prow
+  if [[ -z "${PROJECT_ID:-}" && -n "${PROW_JOB_ID:-}" ]]; then
+      PROJECT_ID="$(boskos_acquire verification-e2e-project)"
+      trap "boskosctl_wrapper release --name \"${PROJECT_ID}\" --target-state dirty >&2" EXIT
+      export PROJECT_ID
+  fi
+
+  ${ROOT}/scripts/terraform.sh init
+
+  export_terraform_output db_apikey_database_key_secret E2E_DB_APIKEY_DATABASE_KEY
+  export_terraform_output db_apikey_signature_key_secret E2E_DB_APIKEY_SIGNATURE_KEY
+  export_terraform_output db_conn E2E_DB_CONN
+  export_terraform_output db_encryption_key_secret E2E_DB_ENCRYPTION_KEY
+  export_terraform_output db_name E2E_DB_NAME
+  export_terraform_output db_password E2E_DB_PASSWORD
+  export_terraform_output db_user E2E_DB_USER
+  export_terraform_output db_verification_code_key_secret E2E_DB_VERIFICATION_CODE_DATABASE_KEY
+  export_terraform_output apiserver_urls[0] E2E_APISERVER_URL
+  export_terraform_output adminapi_urls[0] E2E_ADMINAPI_URL
+  export E2E_DB_PASSWORD="secret://${E2E_DB_PASSWORD}"
+
+  export DB_SSLMODE=disable
```

Comment:
Shouldn't need this. If you do, we have a bug somewhere 😄

Reply:
Yup, you're right! Had some problems and now it's working.